osf/models/mixins.py

```diff
@@ -2041,6 +2041,8 @@ def check_spam(self, user, saved_fields, request_headers):
             return False
         if user.spam_status == SpamStatus.HAM:
             return False
+        if request_headers['Host'].startswith('admin') or ':8001' in request_headers['Host']:
```

Comment:
Just to verify, you're sure that the Host doesn't have the protocol attached to it, so it'll definitely start with admin?

Reply:
I double checked and yes, the protocol is removed from Host.